removeFromArray/removeFromArray.js

```diff
@@ -2,4 +2,31 @@ const removeFromArray = function() {
 
 }
 
+/* Try without using forEach()
+
+const removeFromArray = function(array, ...toRemove) {
+    let testArray = [the array to be passed in];
```

Comment:
You don't need to define a new variable. You can access the array passed into the function with the `array` variable defined as a function argument.